main.py

```diff
@@ -25,8 +25,16 @@ def get_updated_counters(tickets):
 
     filename = "tickets.csv"
     data = read_csv(filename)
+
+    last = data.tail(1)
+    differences = [key for key in new_data.keys() if key != 'utc_epoch_time' and (last[key] != new_data[key]).any()]
+
+    # Append the current row anyhow to move the graph forward
     data = data.append(DataFrame(new_data), ignore_index=True)
-    data.to_csv(filename, index=False)
+
+    if differences:
+        data.to_csv(filename, index=False)
+
```

Comment:
The reason for always writing datapoints is because otherwise slopes would be altered as data between datapoints that live far apart would be interpolated over the whole time range.

That could be solved in code, but we should not spend that effort I think. Adding it to the csv file will not make it gigabytes of data, so it will be okay.

Could you revert this one section? Then I can merge the rest immediately.

Reply:
I've added a flag to stackplot so it no longer interpolates. This should resolve it too while still getting better svg file sizes